src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,105 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.ui.Ui;
+
+
+/**
+ * Represents a command to add tasks to the task list in the Duke application.
+ */
+public class AddCommand extends Command {
+    private String fullCommand;
+    private char type;
+
+    /**
+     * Constructs an AddCommand object with full command and task type.
+     *
+     * @param fullCommand The full command input provided by the user.
+     * @param type The type of task being added ('T' for Todo, 'D' for Deadline, 'E' for Event).
+     */
+    public AddCommand(String fullCommand, char type) {
+        this.fullCommand = fullCommand;
+        this.type = type;
+    }
+
+    /**
+     * Executes the add task command, adding the specified task to the task list and updating the storage.
```

Comment:
Love the Javadocs! They're well explained, illustrating what your code does well